solutions/security/detect-and-alert/create-manage-shared-exception-lists.md

```diff
@@ -4,15 +4,155 @@ mapped_urls:
   - https://www.elastic.co/guide/en/serverless/current/security-shared-exception-lists.html
 ---
 
-# Create and manage shared exception lists
+# Create and manage shared exception lists [shared-exception-lists]
 
-% What needs to be done: Align serverless/stateful
+Shared exception lists allow you to group exceptions together and then apply them to multiple rules. Use the Shared Exception Lists page to set up shared exception lists.
 
-% Use migrated content from existing pages that map to this page:
+::::{note}
+Exception lists created in 8.5 and earlier become shared exception lists in 8.6 or later. You can access all shared exception lists from the Shared Exception Lists page.
+::::
```

Comment:
This note is only applicable to ESS and is only relevant for users who are upgrading from 8.5 -> 8.6 or later. Might need to add this to the [8.6.x upgrade docs](https://www.elastic.co/guide/en/security/8.6/upgrade-intro.html) later.